package.json

```diff
@@ -33,6 +33,7 @@
     "@zendeskgarden/container-utilities": "0.5.4",
     "@zendeskgarden/css-bedrock": "8.0.1",
     "@zendeskgarden/eslint-config": "15.0.0",
+    "@zendeskgarden/react-accordions": "8.19.0",
```

Comment:
Please note, the dependency also needs to be added under the codesandbox config.